### PR TITLE
Base scrapper-API

### PR DESCRIPTION
**Pull request by skhortiuk**
Base scrapper-API

### CODE REVIEW

File: app/options.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+from tornado.options import define, parse_config_file
+
+
+__all__ = ["load_conf", ]
+
+
+def load_conf(config_file=None):
+    # Todo: provide docstring
+    if config_file:
+        parse_config_file(config_file)
```
Comment (rostIvan) [inline app/options.py]: I'm not sure, but first you probably need to `define` those config variables, and then parse them

File: app/main.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        'handlers': urls,
+        'debug': True
+    }
+    return Application(**settings)
+
+
+if __name__ == '__main__':
+    _make_app()
+    load_conf()
+    app = _make_app()
```
Comment (rostIvan) [inline app/main.py:26]: Twice invoke `_make_app()` function, is it ok?

File: app/main.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+from tornado.web import Application
+
+from app.api.urls import urls as api_urls
+from app.options import load_conf
+
+
+def _make_app():
+    # Todo: provide docstring
+    urls = []
+    urls.extend(api_urls)
```
Comment (rostIvan) [inline app/main.py]: It can to look like:
```python
urls = [*api_urls]
```

**Review by rostIvan — APPROVED** [review]

File: app/scraper/loader.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    return await load_page(method='POST', body=body)
+
+
+async def load_teachers_or_groups(query='', faculty='0', teachers=False):
+    # Todo: provide docstring
+    if teachers:
+        api_code = options.teachers_api_code
+    else:
+        api_code = options.groups_api_code
+
```
Comment (rostIvan) [inline app/scraper/loader.py:58]: Can be replaced with
```python
api_code = options.teachers_api_code if teachers else options.groups_api_code
```

File: app/scraper/parser.py [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+def _parse_options_list(options):
+    # Todo: provide docstring
+    faculties_list = []
+    for option in options:
+        faculties_list.append({
+            'name': option.text,
+            'code': int(option['value'])
+        })
+
+    return faculties_list
```
Comment (rostIvan) [inline app/scraper/parser.py:175]: Try it. Maybe works the same
```python
return [{'name': option.text, 'code': int(option['value'])} for option in options]
```

File: app/scraper/loader.py (hunk 4 above)
Comment (skhortiuk) [reply app/scraper/loader.py:58]: Looks like this line is longer than 79 characters.

File: app/scraper/parser.py (hunk 5 above)
Comment (skhortiuk) [reply app/scraper/parser.py:175]: Looks like it'll make the code more difficult to understand. And also do not forget the max line length is 79 characters

File: app/scraper/loader.py (hunk 4 above)
Comment (rostIvan) [reply app/scraper/loader.py:58]: Yep, leave it as it was

File: app/scraper/parser.py (hunk 5 above)
Comment (rostIvan) [reply app/scraper/parser.py:175]: 79 len usually fix by that way
```python
return [{'name': option.text, 'code': int(option['value'])}
         for option in options]
```
Maybe really more difficult to understand, but for me it's ok.
Doesn't matter, you can to skip it.

**Review by IvanYukish — APPROVED** [review]

**Pull request merged**